cypress/integration/PopupStack.spec.ts

```diff
@@ -1,14 +1,25 @@
 import * as h from '../helpers';
 import {queries} from '@testing-library/dom';
 
+const getStackedParent = (element: HTMLElement): HTMLElement | null => {
+  if (element.style.zIndex !== '') {
+    return element;
+  } else {
+    if (element.parentElement) {
+      return getStackedParent(element.parentElement);
+    } else {
+      return null;
+    }
+  }
+};
```

Comment:
This function is resilient to how many DOM elements there are between the element and the stacked parent. As long as `z-index` is set somewhere, we'll compare it.